odd-platform-api/src/test/java/org/opendatadiscovery/oddplatform/repository/ReactiveDataQualityRunsRepositoryTest.java

```diff
@@ -0,0 +1,214 @@
+                        .block());
+
+        dataQualityTestAssertion(testResults);
+    }
+
+    @Test
+    @DisplayName("Test get Monitored Tables")
+    public void testGetMonitoredTables() {
+        final int numberOfNotMonitoredTables = 5;
+        IntStream.range(0, 5)
+                .forEach(i -> createHollowDataSentEntityTable());
+
+        final TablesDashboard tablesDashboard =
+                tablesDashboardMapper.mapToDto(List.of(), repository.getMonitoredTables()
+                        .collectList()
+                        .block());
+
+        assertEquals(numberOfNotMonitoredTables, tablesDashboard.getMonitoredTables().getNotMonitoredTables());
+        assertEquals(NUMBER_OF_ASSERTION + NUMBER_OF_SCHEMA_CHANGES
+                        + NUMBER_OF_COLUMN_ANOMALY + NUMBER_OF_FRESHNESS_ANOMALY + NUMBER_OF_VOLUME_ANOMALY,
+                tablesDashboard.getMonitoredTables().getMonitoredTables());
+    }
+
+    private void generateTestsData() {
+        ReactiveDataQualityRunsRepositoryTest.EXPECTED_VALUES
+                .forEach((key, value) -> generateSpecificTestData(key, value.getLeft(), value.getRight()));
+    }
+
+    private void generateSpecificTestData(final DataQualityCategory category,
+                                          final List<DataEntityRunStatus> statuses,
+                                          final int numberOfTests) {
+        IntStream.range(0, numberOfTests)
+                .mapToObj(value -> createHollowDataSentEntityTable())
+                .forEach(hollowEntity -> {
+                    statuses.forEach(status -> {
+                        final DataEntityPojo assertion = createDataQualityEntityWithCategory(category);
+                        dataQualityTestRelationRepository.createRelations(List.of(
+                                new DataQualityTestRelationsPojo()
+                                        .setDataQualityTestOddrn(assertion.getOddrn())
+                                        .setDatasetOddrn(hollowEntity.getOddrn())
+                        )).block();
+                        createTaskRun(assertion, status);
+                    });
+                });
+    }
+
+    private DataEntityPojo createHollowDataSentEntityTable() {
+        return dataEntityRepository
+                .create(new DataEntityPojo().setOddrn(UUID.randomUUID().toString())
+                        .setTypeId(DataEntityTypeDto.TABLE.getId()))
+                .block();
+    }
+
+    private DataEntityPojo createDataQualityEntityWithCategory(final DataQualityCategory category) {
+        final JSONB jsonb = JSONB.jsonb(
+                DATA_QUALITY_TEST_ATTRIBUTE.replace("{type}", category.name())
+        );
+
+        final List<DataEntityPojo> dataEntityPojos = List.of(new DataEntityPojo()
+                .setOddrn(UUID.randomUUID().toString())
+                .setTypeId(DataEntityTypeDto.JOB.getId())
+                .setSpecificAttributes(jsonb));
```

Comment:
i don't mind but it feels a little bit counter-intuitive. Here we are creating the list of single element. It doesn't feel like there is going to be a case in that particular place code when we get more than one elem list. So what is point to wrap it into a List? Can we do something like this:
```suggestion
        final DataEntityPojo dataEntityPojo = new DataEntityPojo()
                .setOddrn(UUID.randomUUID().toString())
                .setTypeId(DataEntityTypeDto.JOB.getId())
                .setSpecificAttributes(jsonb);
```

Reply:
Sure, for this repo it will be better.

Reply:
done